frontend/src/components/common/sample-csv/SampleCsv.tsx

```diff
@@ -39,11 +41,19 @@ const SampleCsv = ({
         ...without(variableHeaders, ...fixedHeaders),
       ]
 
+      const variableRow = Array(headers.length - 1).fill('abc')
```

Comment:
Am I correct to say that using `abc` is acceptable here because we don't have default parameters for all the different templates' fields?

Reply:
While I'm not able to know what the original committer was thinking, I would believe this is better than seemingly realistic values. Otherwise, might forget to change. As for abc vs some other placeholder value, I prefer something more eye-catching like TEST. But, out of scope of this task/PR.

Reply:
I might have misunderstood your qn, I'll just add on to see if this lands:

I only want to add default params for `language` and nothing else. Since the original behaviour uses `abc`, I'll do the same.